pkg/sqs/SqsMessage.php

```diff
@@ -113,6 +119,21 @@ public function getHeader(string $name, $default = null)
         return array_key_exists($name, $this->headers) ? $this->headers[$name] : $default;
     }
 
+    public function setAttributes(array $attributes): void
+    {
+        $this->attributes = $attributes;
+    }
+
+    public function getAttributes(): array
+    {
+        return $this->attributes;
+    }
+
+    public function getAttribute(string $name, $default = null)
+    {
+        return array_key_exists($name, $this->attributes) ? $this->attributes[$name] : $default;
```

Comment:
Wouldn't `isset` be sufficient? It is said it's a lot faster than `array_key_exists` with the difference that it will report `null` values as not set.

https://ilia.ws/archives/247-Performance-Analysis-of-isset-vs-array_key_exists.html
http://thinkofdev.com/php-fast-way-to-determine-a-key-elements-existance-in-an-array/

At least it was in 2012, but I don't think this changed.

Reply:
sure there are better ways now, `return $this->attributes[$name] ?? $default`, but i was copying the current style of the code base.